doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-RPi/index.md

```diff
@@ -5,4 +5,40 @@ permalink: /:path/
 sort_idx: 60
 ---
 
-SDK integration instructions for select platforms
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The guide provides information on how to **cross-compile** C SDK for Raspberry PI.
+Alternatively, you can build the Kaa C endpoint SDK right on the Raspberry Pi board.
+Refer to [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/) for further details.
+
+# Install dependencies
+
+**The further instructions must be executed on the host machine.**
+
+**Verified against:**
+
+ - **Host OS:** **TODO [KAA-1198](http://jira.kaaproject.org/browse/KAA-1198): Verify agains Ubuntu 14.04 32-bit LTS**
+ - **Target OS:** Raspbian Jessie (2015-11-21)
+
+1. Install build prerequisites
+
+        sudo apt-get install cmake build-essentials
+
+1. Install toolchain
+
+        mkdir rpi_root && cd rpi_root
+        git clone https://github.com/raspberrypi/tools.git
+        export PATH=$(pwd)/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian/bin:PATH
```

Comment:
For the 64-bit OS, should be used another compiler path, I think it'd be useful to be mentioned: `export INSTALLDIR=$RPI_ROOT/tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/arm-linux-gnueabihf/libc/usr`